_includes/utilities/responsive-variants.html

```diff
@@ -63,21 +63,29 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
     </tbody>
   </table>
 
-  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying any settings stored as a map, you need to set <em>all</em> elements of the map, not just the element you wish to change.</p>
+  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. Here is a sample configuration:</p>
 
-  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2 margin-bottom-2">
+  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2 margin-bottom-2 bg-white">
+    <h4 class="font-lang-2xs margin-top-0 margin-bottom-05">Example</h4>
     <pre class=" font-mono-xs margin-0 padding-0 bg-transparent">
 @use "uswds-core" with (
   $background-color-settings: (
-      output: true,
-      responsive: false,
-      active: false,
-      focus: false,
-      hover: true,
-      visited: false,
-    )
+    responsive: true
+  )
 );
-</pre>
+    </pre>
+
+    <h4 class="font-lang-2xs margin-top-0 margin-bottom-05">Output</h4>
+    <pre class=" font-mono-xs margin-0 padding-0 bg-transparent">
+$background-color-settings-complete: (
+  output: true,
+  responsive: true,
+  active: false,
+  focus: false,
+  hover: true,
+  visited: false,
+)
+    </pre>
```

Comment:
I originally liked it because it added a contextual clue in that changing `$color-settings.responsive` actually looks like

```scss
$color-settings: (
   responsive: true;
)
```

and because we have an example on the current iteration of this section

![image](https://github.com/uswds/uswds-site/assets/25211650/adff495b-e90f-4613-a030-e28e5fb937f7)


_but_ I can also see that it may seem redundant with the new section added to the settings page. Happy to remove if we're confident it's the best move!

If we decide to keep it, I can make it match the change to the settings page example by removing Output and not referring to `$background-color-settings-complete`